records_mover/creds/base_creds.py

```diff
@@ -129,20 +133,24 @@ def default_gcs_client(self) -> Optional['google.cloud.storage.Client']:
             return self.__default_gcs_client
         try:
             import google.cloud.storage  # noqa
-        except ModuleNotFoundError:
-            logger.debug("google.cloud.storage not installed",
-                         exc_info=True)
+        except ModuleNotFoundError as e:
+            logger.warning(f"google.cloud.storage not installed: {e}")
+            logger.debug("Details:", exc_info=True)
             self.__default_gcs_client = None
             return self.__default_gcs_client
 
         try:
-            self.__default_gcs_client = google.cloud.storage.Client(credentials=gcs_creds)
+            other_args = {}
+            if 'GCP_PROJECT' in os.environ:
+                other_args['project'] = os.environ['GCP_PROJECT']  # TODO: Mention in CONFIG.md
```

Comment:
Uncompleted punchlist item detected--consider resolving or moving this to your issue tracker